nansat/tests/test_exporter.py

```diff
@@ -15,11 +15,13 @@
 import os
 import sys
 import json
+import time
 import logging
 import unittest
 import warnings
 import datetime
 import tempfile
+from pprint import pprint
```

Comment:
Is that so necessary now? I think, if you find it's good then you should do it generally in development and make it a standard for all print funcs.

Reply:
That I can remove. that was rather for debugging of metadata.